spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/adapters/loadbalancer/RetryAwareLoadBalancer.java

```diff
@@ -46,19 +51,24 @@ public class RetryAwareLoadBalancer implements ReactorServiceInstanceLoadBalance
 
   private final LoadBalancerProperties loadBalancerProperties;
 
+  private final LoadBalanceHandler loadBalanceHandler;
+
   private final Map<String, ReactorServiceInstanceLoadBalancer> loadBalancers = new ConcurrentHashMap<>();
 
   public RetryAwareLoadBalancer(ObjectProvider<ServiceInstanceListSupplier> serviceInstanceListSupplierProvider,
-      String serviceId, LoadBalancerProperties loadBalancerProperties) {
+      String serviceId, LoadBalancerProperties loadBalancerProperties, LoadBalanceHandler loadBalanceHandler) {
     this.serviceInstanceListSupplierProvider = serviceInstanceListSupplierProvider;
     this.serviceId = serviceId;
     this.loadBalancerProperties = loadBalancerProperties;
+    this.loadBalanceHandler = loadBalanceHandler;
   }
 
   @Override
   @SuppressWarnings("rawtypes")
   public Mono<Response<ServiceInstance>> choose(Request request) {
     InvocationContext context = InvocationContextHolder.getOrCreateInvocationContext();
+    String rule = loadBalanceHandler.getActuator(convert(request))!=null?loadBalanceHandler.getActuator(convert(request)).getRule():LoadBalancerProperties.RULE_ROUND_ROBIN;
```

Comment:
1. format code
2. do not put `loadBalanceHandler.getActuator(convert(request))` twice